spec/controllers/users_controller_spec.rb

```diff
@@ -36,6 +37,37 @@
       it { expect(response).to redirect_to log_in_path }
     end
 
+    context "sending an email" do
+      after { ActionMailer::Base.deliveries.clear }
+
+      it "sends the email" do
+        post :create, params: { user: Fabricate.attributes_for(:user) }
+        expect(ActionMailer::Base.deliveries.size).to_not eq(0)
+      end
+
+      it "has the correct message subject" do
+        user_params = Fabricate.attributes_for(:user)
+        post :create, params: { user: user_params }
+        message = ActionMailer::Base.deliveries.last
+        name = user_params[:full_name]
+        expect(message.body.raw_source).to include(name)
```

Comment:
Good point. I'm guessing the video checks the subject, and my original test checked the body. I changed it to subject, thanks!